### DISCUSSION TITLE
GitHub Documentation Basic Writing and Formatting Currently Used For Oqtane.Docs

### DISCUSSION THREAD

**Discussion by thabaum** [oqtane/oqtane.docs#37] [General]
Anyone who would like to contribute to Oqtane.Docs I recommend taking some time and review [GitHub Documentation Basic Writing and Formatting](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) to help produce quality pull requests.

**Comment by iJungleboy** [comment]
Please note that for the docs we use DocFx which uses **CommonMark** and a few extensions.

For example, you can also create diagrams using Mermaid.

The details are here: https://dotnet.github.io/docfx/docs/markdown.html
